CHANGELOG.rst

```diff
@@ -91,9 +99,6 @@ Changed
   (default behavior) or in a temporary directory by specifying the
   ``save_to_default_model_directory`` field in the training request.
 
-[1.3.4]
```

Comment:
what's this about? there's two fixed sections now for 1.3.4... also how was this even passing the build?!

Reply:
This was introduced with 1.3.7, so I'd assume the "fixed" below belongs into 1.3.7 - will move it there

Reply:
actually judging from https://github.com/RasaHQ/rasa/pull/4511, it should just be part of 1.3.4 - not sure why it didn't end up in the right section